### PR TITLE
fix: add the db information in the dml message

### PR DESCRIPTION
**Pull request by SimFG**
- issue: #37966

### CODE REVIEW

**Comment by SimFG** [comment]
rerun cpp-unit-test

**Comment by SimFG** [comment]
rerun go-sdk

File: internal/proxy/msg_pack.go [hunk 1]
```diff
@@ -58,6 +58,8 @@ func genInsertMsgsByPartition(ctx context.Context,
 			),
 			CollectionID:   insertMsg.CollectionID,
 			PartitionID:    partitionID,
+			DbName:         insertMsg.DbName,
+			DbID:           insertMsg.DbID,
```
Comment (xiaofan-luan) [inline internal/proxy/msg_pack.go:62]: what do we need for dbID?

**Pull request merged**